manifests/j/JamesCJ60/Universalx86TuningUtility/Beta/2.2.13/JamesCJ60.Universalx86TuningUtility.Beta.installer.yaml

```
@@ -0,0 +1,25 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: JamesCJ60.Universalx86TuningUtility.Beta
+PackageVersion: 2.2.13
+InstallerLocale: en-US
+InstallerType: msi
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- universalx86tuningutility
+ProductCode: '{46B5637C-C030-4589-B644-42558B303E53}'
+ReleaseDate: 2024-07-13
+AppsAndFeaturesEntries:
+- UpgradeCode: '{300E743A-4C69-440D-9425-14C446AB2658}'
+Installers:

```

```suggestion
Installers:
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.DotNet.Runtime.7
     - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
Installers:
```
Apologies - I forgot to click "Start a review".